### ISSUE TITLE
Problem doing pyoxidizer build with linux musl x86_64 pyembed

### ISSUE THREAD

**Issue by aebm** [indygreg/PyOxidizer#382]
The command with the problem

```
pyoxidizer build --target-triple x86_64-unknown-linux-musl
```

Part of the output mentioning that pyembed has a wrong version it has pyembed=0.13.2 and it expects 0.13.0, 0.12.0, 0.11.0, ...

```
#20 36.93 libpythonXY created                                                                                                                                                   
#20 37.26 building with Rust 1.51.0                                                                                                                                             
#20 38.46     Updating crates.io index                                                                                                                                          
#20 39.84 error: failed to select a version for the requirement `pyembed = "^0.13.2"`                                                                                           
#20 39.84 candidate versions found which didn't match: 0.13.0, 0.12.0, 0.11.0, ...                                                                                              
#20 39.84 location searched: crates.io index                                                                                                                                    
#20 39.84 required by package `restore-redis v0.1.0 (/tmp/pyoxidizeruJK7uG/restore-redis)`                                                                                      
#20 39.87 error[PYOXIDIZER_PYTHON_EXECUTABLE]: adding PythonExecutable to FileManifest                                                                                          
#20 39.87                                                                                                                                                                       
#20 39.87 Caused by:                                                                                                                                                            
#20 39.87     0: building Python executable                                                                                                                                     
#20 39.87     1: building executable with Rust project                                                                                                                          
#20 39.87     2: command ["/root/.cargo/bin/cargo", "build", "--target", "x86_64-unknown-linux-musl", "--target-dir", "/tmp/pyoxidizeruJK7uG/build/target", "--bin", "restore-redis", "--no-default-features", "--features", "build-mode-prebuilt-artifacts cpython-link-unresolved-static global-allocator-jemalloc allocator-jemalloc"] exited with code 101  
#20 39.87    --> ./pyoxidizer.bzl:273:5                                                                                                                                         
#20 39.87     |                                                                                                                                                                 
#20 39.87 273 |     files.add_python_resource(".", exe)                                                                                                                         
#20 39.87     |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ to_file_manifest()                                                                                                      
#20 39.87                                                                                                                                                                       
#20 39.87                                                                      
```

**Comment by indygreg** [comment]
Thanks for reporting.

I've been tweaking the release automation and it seems I found a bug where the versions of PyOxidizer and the `pyembed` crate can get out of sync. This will need a new release to fix. In the meantime, you can try installing version `0.13.0`. My goal is to have `0.14` out by the end of the weekend.

**Comment by indygreg** [comment]
I published a version 0.13.2 of the pyembed crate so existing 0.13.2 installs will work. I also changed the release automation so this hopefully shouldn't happen again.

I don't think there's anything more to be done here, so I'm calling this closed.
